consensus/consensus-types/src/block_data.rs

```diff
@@ -148,6 +148,7 @@ where
             LedgerInfoWithSignatures::new(
                 LedgerInfo::new(ancestor, HashValue::zero()),
                 BTreeMap::new(),
+                BTreeMap::new(),
```

Comment:
I am not sure we need to aggregate markers in the QC. 

It seems we may want to aggregate endorsements on a target block, coming from votes on potentially different blocks, whose markers endorse the target.

Reply:
The current LibraBFT implementation does not use threshold sig. Instead, QC stores 2f+1 signatures in the BTreeMap struct. I am doing similar things for the marker, but the current implementation is not secure (markers are not signed).